spec/migrations/20190327132620_dialog_field_load_values_on_init_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_migration
+
+describe DialogFieldLoadValuesOnInit do
+  let(:dialog_field_stub) { migration_stub(:DialogField) }
+
+  migration_context :up do
+    let (:show_button) { nil }
```

Comment:
I don't think this is needed.

Reply:
It was, `undefined local variable or method `show_button' for #<RSpec::ExampleGroups::DialogFieldLoadValuesOnInit::Up:0x0000000009226718>` otherwise.